paprika/simulate.py

```diff
@@ -0,0 +1,160 @@
+import numpy as np
+import subprocess as sp
+import logging as log
+import os as os
+import parmed as pmd
+from collections import OrderedDict
+
+
+def _write_input_file(filename, input_dict, title='Input.'):
+    """
+    Process a dictionary to generate a pmemd input file.
+    """
+
+    log.debug('Writing {}'.format(filename))
+    with open(filename, 'w') as f:
+        f.write("{}\n".format(title))
+        for namelist in ['cntrl','ewald']:
+            if input_dict[namelist] is not None:
+                f.write(" &{}\n".format(namelist))
+                for key,val in input_dict[namelist].items():
+                    if val is not None:
+                        f.write("  {:15s} {:s},\n".format(key+' =',str(val)))
+                f.write(" /\n")
+        if input_dict['cntrl']['nmropt'] == 1:          ### Need to check if this exists first!!
+            if input_dict['wt'] is not None:
+                for line in  input_dict['wt']:
+                    f.write(" "+line+"\n")
+            f.write(" &wt type = 'END', /\n")
+            if input_dict['DISANG'] is not None:
+                f.write("DISANG = {}\n".format(input_dict['DISANG']))
+                f.write("LISTOUT = POUT\n\n")
+        if input_dict['GROUP'] is not None:
+            f.write("{:s}".format(input_dict['GROUP']))
+
+
+
+class pme_simulation():
+    """
+    All PME/PBC simulation methods.
+    """
+
+    def __init__(self):
+        self.todo = 'todo'
+
+
+class gb_simulation():
+    """
+    All GB simulation methods.
+    """
+
+    def __init__(self):
+
+        
+        self.path = '.'
+        self.prmtop = 'prmtop'
+        self.inpcrd = 'inpcrd'
+        self.job = {}
```

Comment:
Comment these (at minimum first and last one). Is `path` where the simulation is going to write output? Where it will look for the `prmtop` or something else? Also what is `job` doing?

Reply:
Okay, I understand job below. Still comment.